ts/Extensions/Data.ts

```diff
@@ -284,7 +284,13 @@ class Data {
     public alternativeFormat?: string;
     public chart: Chart;
     public chartOptions: Partial<Options>;
-    public columns?: Array<Array<DataValueType>>;
+    public columns?: Array<Array<DataValueType> & {
+        isDatetime?: boolean;
+        isNumeric?: boolean;
+        mixed?: boolean;
+        name?: string;
+        unsorted?: boolean;
+    }>;
```

Comment:
Now thinking about this... How about creating an interface `DataColumnsArray extends Array<Array<DataValueType>` then you can skip als these reference hacks, that might confuse some.